### DISCUSSION TITLE
Zekaküp konsey oylaması sorusu

### DISCUSSION THREAD

**Discussion by sonsuzus** [sonsuzus/sonsuzus#149] [Zeka Sorusu]
Zekaküp konseyinde 200 üyenin katıldığı bir oylama yapılmıştır. Önerge lehinde oy kullanmış olanların dörtte biri aleyhte oy kullanmış olsaydı, Önerge yine de kabul edilecekti.
Oysa lehte oy kullananların üçte biri aleyhte oy kullanmış olsaydı, önerge reddedilecekti.
Önerge lehinde en az kaç oy kullanılmış olabilir? (Çekimser oy yoktur.)

**Comment by Sadik1Dev** [comment]
x lehte y aleyhte oy verenler olsun
x+y=200
x>100
x>y
1. şart: x-x/4>y+x/4  buradan 3x/4>4y+x/4 sadeleştirme 2x>4y ve x>2y
2.  şart x-x/3<y+x/3 buradan 2x/3<3y+x/3 sadeleştirme x<3y
birleştirirsek 3y>x>2y x yerine 200-y yazalım 
3y>200-y>2y buradan 4y>200>3y elde ederiz bu eşitsizliği sağlayan en büyük y değerini bulursan en az lehte oy veren x leri buluruz.

y en fazla 66,66 devirli olabilir tam sayı olarak en fazla 66 olacağından x=200-y=200-66=134

en az lehte oy veren kişi sayısı: 134'tür.
